ckanext/ckanext-sixodp_ui/ckanext/sixodp_ui/templates/package/resource_read.html

```diff
@@ -22,9 +22,10 @@
           <div class="heading-container">
             {% block resource_read_title %}{% endblock %}
             {% block page_subheading %}
-              <h2 class="page-subheading">
-                <span>{{ h.render_datetime(h.get_created_or_updated(res), '%-d.%-m.%Y') }}</span>
-              </h2>
+              {% set resource_date = h.get_created_or_updated(res) %}
+              {% if resource_date %}
+                <div class="page-subheading">{{ h.render_datetime(resource_date, '%-d.%-m.%Y') }}</div>
```

Comment:
hmm, will this work? I replaced the heading element with div but added the same class "page-subheading"?